### PR TITLE
memory cleanup

### PR DESCRIPTION
**Pull request by heythisisnate**

### CODE REVIEW

File: src/lfs/application.lua [hunk 1]
```diff
@@ -6,11 +6,21 @@ local actuators = require("actuators")
...
+  -- handle strings or numbers
+  --return zoneMap[zone] or zoneMap[tonumber(zone)]
+  if zone == 1 or zone == '1' then return 1 end
+  if zone == 2 or zone == '2' then return 2 end
+  if zone == 3 or zone == '3' then return 5 end
+  if zone == 4 or zone == '4' then return 6 end
+  if zone == 5 or zone == '5' then return 7 end
+  if zone == 6 or zone == '6' then return 9 end
+  if zone == "out" then return 8 end
+end
```
Comment (heythisisnate) [inline src/lfs/application.lua:23]: ✨

File: src/lfs/server_receiver.lua [hunk 2]
```diff
@@ -17,19 +17,17 @@ local function httpReceiver(sck, payload)
   end
   collectgarbage()
 
-  local request = require("httpd_req")(payload)
-  local response = require("httpd_res")()
+  request = require("httpd_req")(payload)
+  response = require("httpd_res")()
```
Comment (heythisisnate) [inline src/lfs/server_receiver.lua:21]: curious what does removing the `local` declaration actually do?

Comment (h2zero) [reply src/lfs/server_receiver.lua:21]: This makes the variables global so they can be accessed in the offline callback.

File: src/lfs/application.lua (hunk 1 above)
Comment (h2zero) [reply src/lfs/application.lua:23]: :laughing:

**Pull request merged**